### DISCUSSION TITLE
Bare metal + existing config

### DISCUSSION THREAD

**Discussion by Ziomal12** [wg-easy/wg-easy#647] [Q&A] [answered]
Hello,
I wanna ask if there's a way to install this on ubuntu 22.10 LTS directly (without docker) and have it control already established network.

**Comment by eandersons** [comment]
[Search](/wg-easy/wg-easy/issues?q=is%3Aissue+without+docker) is your friend. There already are a couple of issues regarding this. For example, latest comments in #601 contain some possible hints.

**Comment by pheiduck** [comment, marked as answer]
Wiki post have been updated due the old link for the systemd service wasn't working anymore.
https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker
